app/models/annotation/Annotation.scala

```diff
@@ -178,16 +178,24 @@ class AnnotationDAO @Inject()(sqlClient: SQLClient, annotationLayerDAO: Annotati
 
   private def listAccessQ(requestingUserId: ObjectId): String =
     s"""
-       |(visibility = '${AnnotationVisibility.Internal}' and (select _organization from webknossos.teams where webknossos.teams._id = _team)
-       |            in (select _organization from webknossos.users_ where _id = '${requestingUserId.id}'))
-       |          or _team in (select _team from webknossos.user_team_roles where _user = '${requestingUserId.id}' and isTeamManager)
-       |          or _user = '${requestingUserId.id}'
-       |          or (select _organization from webknossos.teams where webknossos.teams._id = _team)
-       |            in (select _organization from webknossos.users_ where _id = '${requestingUserId.id}' and isAdmin)
-       |""".stripMargin
-
-  override def readAccessQ(requestingUserId: ObjectId) =
-    s"""(visibility = '${AnnotationVisibility.Public}' or ${listAccessQ(requestingUserId)})"""
+        (_user = '${requestingUserId.id}'
+        or ((visibility = '${AnnotationVisibility.Public}' or visibility = '${AnnotationVisibility.Internal}')
+          and _id in
+          (select distinct a._annotation
+           from webknossos.annotation_sharedTeams a
+           join webknossos.user_team_roles t
+           on a._team = t._team
+           where t._user = '${requestingUserId.id}')))
+       """
+
+  override def readAccessQ(requestingUserId: ObjectId): String =
+    s"""    (visibility = '${AnnotationVisibility.Public}'
```

Comment:
Please have the outer brackets on separate lines for readability, everything else is quite parsable :+1: 
```suggestion
    s"""(
             visibility = '${AnnotationVisibility.Public}'
```